src/s3fs_util.cpp

```diff
@@ -1151,6 +1151,10 @@ void show_help (void)
     "   curldbg - put curl debug message\n"
     "        Put the debug message from libcurl when this option is specified.\n"
     "\n"
+    "   cipher_suites - customize TLS cipher suite list\n"
+    "        Customize the list of TLS cipher suites.\n"
+    "        Expects a colon separated list of cipher suite names.\n"
```

Comment:
I've added instructions on the doc (man and command-line usage printout) with a link to the list of cipher suites.